### DISCUSSION TITLE
Using filter with LTAR or Lookup column [Sep 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3741] [Discord Threads]
> 🙋‍♂️ **4oject**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### `nested` is for applying filter on nested level list/data, if you want to filter root level data do following =>  `where=(createdby,eq,firstuser)` (Assuming  `username` is primary value)  
#  
> 👨‍💻 **pranavxc**  
#### if username is not primary value then create a lookup column and use it within where query  
#  
> 👨‍💻 **pranavxc**  
#### like `(lookupColumn,eq,firstuser)`    
#### where `lookupColumn` is a lookup column to `createdBy => username`  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1012858447538172005)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
